tasks/github.rake

```diff
@@ -6,17 +6,19 @@ namespace :github do
   task :generate_batches do
     matrix = eval(File.read('Matrixfile')).freeze # rubocop:disable Security/Eval
 
-    # Uncomment to only runs the specified candidates
-    candidates = [
-      # 'sidekiq', # Connection refused - connect(2) for 127.0.0.1:6379 (RedisClient::CannotConnectError)
-      # 'mongodb', # Connection issue with multi db and process hang
+    # TODO: These are the execptions, find a way to describe those service dependencies in CI using a more generic mechansim.
```

Comment:
A per-group `services: [service1, service2]` key within the `Matrixfile` could be a nice spot to define such dependencies.

This could even generalise to "standard" ones and help get rid of the arbitrary split.

Reply:
Yup, that requires quite some changes to adopt a new data structure. So I am delaying this pain.